### PR TITLE
Latest GraalVM github action changes

### PR DESCRIPTION
**Pull request by msupic**
The changes from this PR are already used in the `micronaut-project-template` graalvm workflows (for example, [graalvm-dev.yml](https://github.com/micronaut-projects/micronaut-project-template/blob/master/.github/workflows/graalvm-dev.yml) is using `build-matrix@latest-graalvm-changes` task). Since all micronaut projects have been synced with `micronaut-project-template` workflows, we can push changes from this branch to the master branch and then modify workflows in the micronaut-project-template (replace `build-matrix@latest-graalvm-changes` with `build-matrix@master`, `pre-build@latest-graalvm-changes` with `pre-build@master`.

**Important**: Since we need to keep the `latest-graalvm-changes` branch until `micronaut-project-template` graalvm workflows start using github actions from the `master` branch, I created `latest-graalvm-changes-2` from `latest-graalvm-changes` so `latest-graalvm-changes` doesn't get deleted when pushing these changes to the `master` branch.

### CODE REVIEW

File: graalvm/build-matrix/action.yml [hunk 1]
```diff
@@ -20,11 +13,10 @@ runs:
         fetch-depth: 0
 
     - name: "Setup GraalVM CE"
-      uses: graalvm/setup-graalvm@v1.0.12
+      uses: graalvm/setup-graalvm@v1
```
Comment (sdelamo) [inline graalvm/build-matrix/action.yml:16]: don't p `@v1` use a fully qualified version number.

File: graalvm/pre-build/action.yml [hunk 2]
```diff
@@ -25,9 +25,9 @@ runs:
         fetch-depth: 0
 
     - name: "Setup GraalVM CE"
-      uses: graalvm/setup-graalvm@v1.0.12
+      uses: graalvm/setup-graalvm@v1
```
Comment (sdelamo) [inline graalvm/pre-build/action.yml:28]: don't p `@v1` use a fully qualified version number.

**Review by sdelamo — CHANGES_REQUESTED** [review]

File: graalvm/build-matrix/action.yml (hunk 1 above)
Comment (msupic) [reply graalvm/build-matrix/action.yml]: Replaced with v1.1.1

File: graalvm/pre-build/action.yml (hunk 2 above)
Comment (msupic) [reply graalvm/pre-build/action.yml]: Replaced with v1.1.1

**Review by sdelamo — APPROVED** [review]

**Pull request merged**